deployment/amplify/staging/amplify/backend/function/echolocatorStagingClients/src/app.js

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
+    http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+*/
+
+
+/* Amplify Params - DO NOT EDIT
+You can access the following resource attributes as environment variables from your Lambda function
+var environment = process.env.ENV
+var region = process.env.REGION
+var authEcholocatorStagingAuthUserPoolId = process.env.AUTH_ECHOLOCATORSTAGINGAUTH_USERPOOLID
+
+Amplify Params - DO NOT EDIT */
+
+var AWS = require('aws-sdk')
+var express = require('express')
+var bodyParser = require('body-parser')
+var awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')
+
+// declare a new express app
+var app = express()
+app.use(bodyParser.json())
+app.use(awsServerlessExpressMiddleware.eventContext())
+
+// Enable CORS for all methods
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Credentials", true)
+  res.header("Access-Control-Allow-Origin", "*")
```

Comment:
Would there be any benefit or harm from setting this more strictly?

Reply:
It would be extra work to figure out, test, and maintain the exact host list to allow. I don't see much of a security benefit upside, considering AWS credential permissions control access.